src/txn/txn.c

```diff
@@ -1218,10 +1241,29 @@ __wt_txn_rollback(WT_SESSION_IMPL *session, const char *cfg[])
 			 * Need to resolve indirect references of transaction
 			 * operation, in case of prepared transaction.
 			 */
-			if (F_ISSET(txn, WT_TXN_PREPARE))
-				WT_RET(__wt_txn_resolve_prepared_op(
-				    session, op, false));
-			else {
+			if (F_ISSET(txn, WT_TXN_PREPARE)) {
+				/*
+				 * Decrement the resolved update count as we
+				 * as we would either resolve at least 1 update
```

Comment:
Same comment as before. `as we` is repeated. 
Since this is a copy of the earlier code, @michaelcahill's comment also applies here. Maybe consider refactoring into a function to have one copy.